pkg/table/context/table.go

```diff
@@ -133,6 +133,19 @@ type MutateContext interface {
 	// which is a buffer for table related structures that aims to reuse memory and
 	// saves allocation.
 	GetMutateBuffers() *MutateBuffers
+
+	// SetExtraIndexKeyPosInfo sets the extra index key pos info.
+	// It's used when the given row is not a full row, but only contains some limit columns
+	SetExtraIndexKeyPosInfo(indexKeyPos map[int64][]int)
+
+	// GetExtraIndexKeyPosInfo gets the extra index key pos info.
+	GetExtraIndexKeyPosInfo(idxID int64) []int
+
+	// ResetExtraInfo resets the extra index key pos info.
+	ResetExtraInfo()
```

Comment:
I think it's better to have a consistent naming.
"ExtraInfo" is a bit confusing. We already call it "ExtraIndexKeyPosInfo", let's just call it "ExtraIndexKeyPosInfo" everywhere.